src/content/TipsAndTricks.js

```diff
@@ -30,21 +30,22 @@ class TipsAndTricks extends React.Component {
                     </p>
 
                     <CodeBlock>
-                        {`def main(args: Array[String]): Int32 & Impure = ...`}
+                        {`def main(): Unit & Impure = ...`}
                     </CodeBlock>
 
                     <p>
-                        That is, the main function (i) takes an array of string arguments, (ii) must return
-                        an <Code>Int32</Code> which represents the exit code, and (iii) must be <Code>Impure</Code>.
+                        That is, the main function (i) must return <Code>Unit</Code>, (ii) must be <Code>Impure</Code>,
+                        and (iii) must be private.
```

Comment:
Are these the correct requirements for the new `main`?

Reply:
It does not have to be private. (We changed all top level to be implicitly public). So if you keep (i) and (ii) its good 👍